submissions/OlStaniev/DOM/script/text-episode.js

```diff
@@ -0,0 +1,135 @@
+const episodes = [{
+    id:'ep1',
+    title:`Episode I`,
+    subTitle:`THE PHANTOM MENACE`,
+    p1:`Turmoil has engulfed the
+    Galactic Republic. The taxation
+    of trade routes to outlying star
+    systems is in dispute.`,
+    p2:`Hoping to resolve the matter
```

Comment:
If the next episode has description in 10 paragraphs?
Maybe create property description which have array of paragraphed.